.changeset/fluffy-taxis-greet.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kubernetes-models/karpenter": major
```

Comment:
This is a minor because there are no breaking changes.

```suggestion
"@kubernetes-models/karpenter": minor
```

Reply:
thanks, applied!